canonical-web-and-design/jaas-dashboard
# Reduce models icon width to 14px

## Done

Reduce models icon width to 14px

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify models icon is 16x14px

## Details

Fixes #306
